S4ModApi/CCustomUi.cpp

```diff
@@ -43,9 +43,9 @@ CCustomUi::CCustomUi(LPCS4CUSTOMUIELEMENT param) :
 	auto mod = param->mod;
 	if (param->szImg) {
 		if (m_flags & S4_CUSTOMUIFLAGS_FROMRES_IMG) {
-			m_hImg = (HBITMAP)LoadImageW(mod, param->szImg, IMAGE_BITMAP, 0, 0, 0);
+			m_hImg = (HBITMAP)LoadImageW(mod, param->szImg, IMAGE_BITMAP, 0, 0, 0 | LR_LOADTRANSPARENT);
```

Comment:
Have you checked whether the **LR_LOADTRANSPARENT** is necessary? Because if I interpret the documentation correctly, it says that it will just do color keying:
> Retrieves the color value of the first pixel in the image and replaces the corresponding entry in the color table with the default window color (COLOR_WINDOW)

source: https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-loadimagea